### DISCUSSION TITLE
Rebuilding manager

### DISCUSSION THREAD

**Discussion by wtl06639** [Security-Onion-Solutions/securityonion#5661] [Unsupported Versions] [answered]
I have a distributed deployment setup. If i were to rebuild the manager is it easy to reconnect all the sensors to the new manager? Or is it easier/cleaner to just reinstall the sensors as well?

**Comment by LS3-Allen** [comment, marked as answer]
In my experience it is much easier to just reinstall the sensors as well.  I tried to reconfigure sensors to connect to a redeployed manager and never could get it working smoothly.
